src/canon/Chemistry.jl

```diff
@@ -28,14 +28,15 @@ using Markdown
 end)
 
 @docapode("Gray-Scott"
-  ,"https://www.google.com"
-  ,"A model of reaction-diffusion"
+  ,""
+  ,"[pearson_complex_1993](@cite)"
```

Comment:
"A model of reaction-diffusion" is good for the short description along with the citation.